packages/electron-builder-squirrel-windows/src/SquirrelWindowsTarget.ts

```diff
@@ -28,34 +50,42 @@ export default class SquirrelWindowsTarget extends Target {
     const setupFile = packager.expandArtifactNamePattern(this.options, "exe", arch, "${productName} Setup ${version}.${ext}")
     const installerOutDir = path.join(this.outDir, `squirrel-windows${getArchSuffix(arch)}`)
     const artifactPath = path.join(installerOutDir, setupFile)
+    const msiArtifactPath = artifactPath.replace(".exe", ".msi")
 
     await packager.info.callArtifactBuildStarted({
       targetPresentableName: "Squirrel.Windows",
       file: artifactPath,
       arch,
     })
 
-    if (arch === Arch.ia32) {
-      log.warn("For windows consider only distributing 64-bit or use nsis target, see https://github.com/electron-userland/electron-builder/issues/359#issuecomment-214851130")
-    }
```

Comment:
Wait, why was this warning log removed?

Reply:
Discussed via discord. 👍

Reply:
Any chance of sharing for those not in that Discord conversation :D?